src/applications/personalization/profile360/containers/PaymentInformation.jsx

```diff
@@ -131,11 +174,16 @@ class PaymentInformation extends React.Component {
     } else {
       const paymentAccount = paymentInformation.responses[0].paymentAccount;
 
+      // Cannot use arrow-function below in jsx attribute to pass
```

Comment:
You can get around the parser error you were probably seeing by wrapping the arrow function in parens as shown in the screenshot. If you do it this way, with an arrow function instead of relying on the `data-profile-section` you can remove the changes to the `ProfileFieldHeader` component. I think that passing the analytics event directly is a better solution because we aren't adding cruft to the DOM that's only used for this one purpose.

![image](https://user-images.githubusercontent.com/20728956/60743420-5b3feb00-9f26-11e9-8be9-e1918859c5a4.png)

Reply:
Dang!  Forgot about that trick!  Thx!